### PR TITLE
105/retrain ner

### PR DESCRIPTION
**Pull request by Samuel-Campbell**
- Retrained the ner with more data
- Added stop words

### CODE REVIEW

File: src/ml_service/preprocessing/French/GlobalVariable.py [hunk 1]
```diff
@@ -13,7 +13,23 @@ def get_path():
 
 def get_stop_words():
     try:
```
Comment (arekmano) [inline src/ml_service/preprocessing/French/GlobalVariable.py]: why do you try/except here?

File: src/ml_service/preprocessing/main.py [hunk 2]
```diff
@@ -12,9 +12,9 @@
     parser = Parser()
     j = 0
     for i in os.listdir(Global.Precedence_Directory):
-        if 'AZ-51210' in i and j < 5:
+        if 'AZ-5120' in i and j < 10:
```
Comment (arekmano) [inline src/ml_service/preprocessing/main.py:15]: what is this magic string changing?

Comment (Samuel-Campbell) [reply src/ml_service/preprocessing/main.py:15]: it's the same string you use to find texts in the decision bank. I'm just experimenting with different texts at random

File: src/ml_service/preprocessing/French/GlobalVariable.py (hunk 1 above)
Comment (Samuel-Campbell) [reply src/ml_service/preprocessing/French/GlobalVariable.py]: removed

File: src/ml_service/preprocessing/French/DecisionParse.py [hunk 3]
```diff
@@ -121,5 +121,5 @@ def __populate_model(self, line):
     # ** This method can be enhanced for better classification
     #    This is just a proof of concept for now
     def __split_sub_sentence(self, sentence):
-        sentence = sentence.replace(',', ".")
+        #sentence = sentence.replace(',', ".")
```
Comment (arekmano) [inline src/ml_service/preprocessing/French/DecisionParse.py:124]: remove unused line?

**Review by arekmano — APPROVED** [review]

**Pull request merged**